dapp/src/utils/image.js

```diff
@@ -0,0 +1,5 @@
+export function adjustSrcOption(src) {
```

Comment:
nit: naming is hard. but wonder if naming this `rootPath` or `basePath` would represent better what this method does.

Reply:
So there is `Find in Files (and replace)` option in Sublime :)

Thanks renamed it to `assetRootPath`